presto-docs/src/main/sphinx/connector/oracle.rst

```diff
@@ -0,0 +1,71 @@
+===============
+Oracle Connector
+===============
```

Comment:
Please update L1, 3, 27, 36 and 62 to fix build failures.

Reply:
done